lib/datura/parser_options/clear_index.rb

```diff
@@ -1,6 +1,6 @@
 module Datura::Parser
   def self.clear_index_params
-    @usage = "Usage: [solr_]clear_index -[options]..."
+    @usage = "Usage: (solr_)clear_index -[options]..."
```

Comment:
Should this one be `(es|solr)_clear_index`?

Reply:
Oops, I guess so.  I kinda thought I had made it the "default" without es but looking back I have not!

Reply:
Ah okay. No problem :)